ProjectSidewalk/SidewalkWebpage
# Fixes wrong face icon

Resolves #3146 

Corrects neutral face SVG element for hover popup.

##### Before/After screenshots (if applicable)
Before: 
![3146_hover_popup_face_icon_bug1](https://github.com/ProjectSidewalk/SidewalkWebpage/assets/106000281/c22b57c0-d870-4a38-9229-c762c9e19cee)
![3146_hover_popup_face_icon_bug2](https://github.com/ProjectSidewalk/SidewalkWebpage/assets/106000281/eb0a7fc0-d654-4b9d-82da-2845eef3f049)

After:
![3146_hover_popup_face_icon_fix1](https://github.com/ProjectSidewalk/SidewalkWebpage/assets/106000281/627abb20-6e51-40af-be6f-84bc2692da69)
![3146_hover_popup_face_icon_fix2](https://github.com/ProjectSidewalk/SidewalkWebpage/assets/106000281/e1a681b9-cc2a-42a3-9ba4-cb10786be784)

##### Testing instructions
1. Navigate to Explore page
2. Create label and select severity 1 in context menu
3. Check hover popup face icon matches with severity face selected

##### Things to check before submitting the PR
